hack/jenkins/e2e.sh

```diff
@@ -110,6 +110,7 @@ GCE_PARALLEL_SKIP_TESTS=(
     "ResourceUsage"
     "SchedulerPredicates"
     "Services.*restarting"
+    "DaemonRestart"
```

Comment:
maybe put this in GCE_FLAKY and GCE_PARALLEL_FLAKY rather than GCE_PARALLEL_SKIP?